### PR TITLE
Handle channel open manually

### PR DESCRIPTION
**Pull request by TonyGiorgio**
This allows others to open a channel to us, useful for LSP's.

A later PR can handle 0 conf LSP stuff.

### CODE REVIEW

File: node-manager/src/event.rs [hunk 1]
```diff
@@ -326,15 +326,43 @@ impl EventHandler {
...
+                    format_args!("EVENT: OpenChannelRequest incoming"),
                     "event",
                     "",
                     0,
                 ));
+
+                match self.channel_manager.accept_inbound_channel(
+                    &temporary_channel_id,
+                    &counterparty_node_id,
+                    0, // we don't track our channels currently
```
Comment (benthecarman) [inline node-manager/src/event.rs:345]: if we accept multiple channels will there be a problem with overlap, should we generate a random number here instead?

**Review by benthecarman — APPROVED** [review]
looks good to me otherwise

File: node-manager/src/event.rs (hunk 1 above)
Comment (TonyGiorgio) [reply node-manager/src/event.rs:345]: I haven't ran into it yet but shouldn't be difficult to already do a random number, will add that.

Comment (TonyGiorgio) [reply node-manager/src/event.rs]: Added random number generation here.

**Review by benthecarman — APPROVED** [review]

**Pull request merged**